.github/workflows/one-click-release.yml

```diff
@@ -0,0 +1,164 @@
+name: One Click Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      channel:
+        required: true
+        type: choice
+        description: Select a channel. For cloud, use main branch only!
+        options:
+          - ga
+          - beta
+          - cloud
```

Comment:
Sorry for the late comment. but if I am not mistaken the correct name of the channel should be `hosting` and not `cloud`